packages/mobile/src/screens/favorites-screen/useCollectionsScreenData.ts

```diff
@@ -117,38 +117,34 @@ export const useCollectionsScreenData = ({
         return []
       }
 
-      return fetchedCollectionIds.filter((collectionId) => {
+      const offlineCollectionsStatus = getOfflineCollectionsStatus(state)
```

Comment:
I'm trying to remember which parts of state here might be best to select from. When we do the initial step where we load the offline data into the store, what do we use to determine which are avail? Is it just a crawl through the directory or do we use the status like you are using here? I wonder if there is a way to add a selector to kinda say "get downloaded collections"?